pkg/controller.v1/mpi/mpijob_controller.go

```diff
@@ -164,13 +161,13 @@ func (jc *MPIJobReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 	// MPIJob needs not service
 	err = jc.ReconcileJobs(mpijob, mpijob.Spec.MPIReplicaSpecs, mpijob.Status, &mpijob.Spec.RunPolicy)
 	if err != nil {
-		logrus.Warnf("Reconcile MPIJob error %v", err)
+		logger.Info("Reconcile MPIJob error", "error", err)
 		return ctrl.Result{}, err
 	}
 
 	t, err := util.DurationUntilExpireTime(&mpijob.Spec.RunPolicy, mpijob.Status)
 	if err != nil {
-		logrus.Warnf("Reconcile MPIJob Job error %v", err)
+		logger.Info("Reconcile MPIJob Job error", "error", err)
```

Comment:
I think we need to replace it with `logger.Error`.

Reply:
@Electronic-Waste fixed ([c6de073](https://github.com/kubeflow/training-operator/pull/2177/commits/c6de073ee0eeb0af37f19a515f7537ac62096f44))